library/HTMLPurifier/AttrDef/HTML/LinkRel.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * Keywords that are body-ok affect whether link elements are allowed in the body.
+ * The body-ok keywords are dns-prefetch, modulepreload, pingback, preconnect, prefetch, preload, prerender,
+ * and stylesheet.
+ *
+ * https://html.spec.whatwg.org/multipage/links.html#body-ok
+ *
+ * @note We cannot use Enum because multiple values are allowed.
+ */
+class HTMLPurifier_AttrDef_HTML_LinkRel extends HTMLPurifier_AttrDef_HTML5_ARel
+{
+    /**
+     * Lookup table for valid values
+     * @var array
+     */
+    protected static $values = array(
+        'dns-prefetch' => true,
```

Comment:
For now allow `stylesheet` only. Other rel values may need some additional logic described in the WHATWG spec (see comment about `preload` value in `testMultiRel()`), which are not implemented here.

The minimum requirement for this library is that purified output has to be spec compliant.